app/models/user.rb

```diff
@@ -2,8 +2,9 @@
 # Fields are all devise settings; most of the methods relate to call list mgmt.
 class User
   include Mongoid::Document
-  include Mongoid::Enum
   include Mongoid::Userstamp::User
+  extend Enumerize
+  extend ActiveModel::Naming
```

Comment:
Ha you made me realize a flaw in my coding is big commits. I usually don't change bad habits until somebody actually points it out. Thanks for the valuable feedback.